www/src/components/TestingGuideline.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import Markdown from 'markdown-to-jsx';
+import { Button, Collapsible, DataTable } from '~paragon-react';  // eslint-disable-line
+import { Close, Check } from '~paragon-icons'; // eslint-disable-line
+
+const TestingGuideline = ({
+  title,
+  href,
+  setIsGuidelinesOnPage,
+  defaultText,
+  events,
+  dataTestId,
+  selectors,
+}) => {
+  const [collapseIsOpen, setCollapseIsOpen] = useState(false);
+  setIsGuidelinesOnPage(true);
```

Comment:
[clarification] Is it OK this happens on every component re-render? Or should it be in a `useEffect` with an empty dependency array?

Reply:
I think the idea of having `useEffect` is better than simple function call. But as I read about setting state to the same value does nothing. Correct me if it is not right. But according to every render function call, yes, `useEffect` is better here